components/webpack/webpack.config.js

```diff
@@ -11,13 +11,14 @@ module.exports = (env, argv) => ({
   resolve: {
     extensions: ['.js', '.tsx', '.ts', '.json'],
     alias: {
-      'bittorrent-tracker': path.resolve(__dirname, '../../node_modules/bittorrent-tracker'),
       'brave-ui': path.resolve(__dirname, '../../node_modules/brave-ui/src'),
-      'dgram': 'chrome-dgram',
-      'dns': path.resolve(__dirname, '../common/dns.ts'),
-      'net': 'chrome-net',
-      'torrent-discovery': path.resolve(__dirname, '../../node_modules/torrent-discovery')
-    }
+      // TODO(feross): Publish chrome-dns and add to "chromeapp" package.json
+      // field so this isn't required anymore
+      dns: path.resolve(__dirname, '../common/dns.ts')
```

Comment:
I'm going to shim `dns` as well, but haven't done that yet. Once I publish `chrome-dns`, I will include it in `webtorrent`'s `"chromeapp"` field in `package.json` and then we can remove this line as well :)